helm/burrow/templates/deployment.yaml

```diff
@@ -18,6 +18,16 @@ spec:
         app: {{ .Values.service.name }}
         visualize: "true" 
     spec:
+      affinity:
+        podAntiAffinity:
```

Comment:
After talking with you and Sorin also, I'm pretty sure we don't need this, the reasons are:
- there is only 1 instance of burrow so pod anti-affinity by instance name won't apply, as there are no instances to separate
- we don't want to scale up the burrow service instances, so there wouldn't be a future use of this configuration

Reply:
👍